app/assets/javascripts/tylium/modules/sidebar.js

```diff
@@ -28,23 +43,25 @@
       this.$navbarBrand.css('padding-left', 0);
       this.$sidebar
         .removeClass('sidebar-expanded')
-        .addClass('sidebar-collapsed')
+        .addClass('sidebar-collapsed');
       this.$viewContent.css({'left': this.$sidebar.css('width'), 'width': 'calc(100vw - ' + this.$sidebar.css('width') + ')'});
 
       this.changeState(false);
     },
     isSidebarOpen: function() {
-      return JSON.parse(localStorage.getItem(this.storageKey))
+      return JSON.parse(localStorage.getItem(this.storageKey));
     },
     open: function() {
       this.$sidebar
         .removeClass('sidebar-collapsed')
-        .addClass('sidebar-expanded')
+        .addClass('sidebar-expanded');
       this.$viewContent.css({'left': this.$sidebar.css('width'), 'width': 'calc(100vw - ' + this.$sidebar.css('width') + ')'});
 
-      var navbarBrandOffset = parseFloat(this.$navbarBrand.css('padding-left').slice(0,-2)) + parseFloat(this.$sidebar.css('width').slice(0,-2) / 1.65);
-      this.$navbarBrand.css('padding-left', navbarBrandOffset);
-      
+      if (window.innerWidth > this.minBreakpoint) {
+        var navbarBrandOffset = parseFloat(this.$navbarBrand.css('padding-left').slice(0,-2)) + parseFloat(this.$sidebar.css('width').slice(0,-2) / 1.65);
+        this.$navbarBrand.css('padding-left', navbarBrandOffset);
+      }
+
```

Comment:
only move the navbar brand (Team/Project name) when the sidebar is expanded on desktop size viewports